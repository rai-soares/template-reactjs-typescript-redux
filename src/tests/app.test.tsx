/* eslint-disable react/react-in-jsx-scope */
import { render, screen } from "@testing-library/react";
import App from "../components/app";

describe("when render app", () => {
  it("should render app", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
