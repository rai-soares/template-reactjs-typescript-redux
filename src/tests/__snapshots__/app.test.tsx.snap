// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when render app should render app 1`] = `
<div>
  <div
    class="sc-bdvvtL csCgPK"
  >
    <header
      class="sc-gsDKAQ ceZCya"
    >
      <a
        class="sc-dkPtRN cynzDA"
        href="/"
      >
        Store
      </a>
      <button
        class="sc-hKwDye hEXOsa"
      >
        <svg
          color="white"
          fill="currentColor"
          height="30"
          stroke="currentColor"
          stroke-width="0"
          style="color: white;"
          viewBox="0 0 16 16"
          width="30"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 1.5A.5.5 0 0 1 .5 1H2a.5.5 0 0 1 .485.379L2.89 3H14.5a.5.5 0 0 1 .491.592l-1.5 8A.5.5 0 0 1 13 12H4a.5.5 0 0 1-.491-.408L2.01 3.607 1.61 2H.5a.5.5 0 0 1-.5-.5zM3.102 4l1.313 7h8.17l1.313-7H3.102zM5 12a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm7 0a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm-7 1a1 1 0 1 1 0 2 1 1 0 0 1 0-2zm7 0a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"
          />
        </svg>
        <span
          class="quantity"
        >
          0
        </span>
        <span
          class="total"
        >
          R$ 
          0,00
        </span>
      </button>
    </header>
    <div
      class="sc-iqseJM cEOUDX"
    >
      <div
        class="sc-dJjYzT gkmPAG"
      />
    </div>
  </div>
</div>
`;

exports[`when render app should render shelf 1`] = `
<div>
  <div
    class="sc-bdvvtL csCgPK"
  >
    <header
      class="sc-gsDKAQ ceZCya"
    >
      <a
        class="sc-dkPtRN cynzDA"
        href="/"
      >
        Store
      </a>
      <button
        class="sc-hKwDye hEXOsa"
      >
        <svg
          color="white"
          fill="currentColor"
          height="30"
          stroke="currentColor"
          stroke-width="0"
          style="color: white;"
          viewBox="0 0 16 16"
          width="30"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 1.5A.5.5 0 0 1 .5 1H2a.5.5 0 0 1 .485.379L2.89 3H14.5a.5.5 0 0 1 .491.592l-1.5 8A.5.5 0 0 1 13 12H4a.5.5 0 0 1-.491-.408L2.01 3.607 1.61 2H.5a.5.5 0 0 1-.5-.5zM3.102 4l1.313 7h8.17l1.313-7H3.102zM5 12a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm7 0a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm-7 1a1 1 0 1 1 0 2 1 1 0 0 1 0-2zm7 0a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"
          />
        </svg>
        <span
          class="quantity"
        >
          0
        </span>
        <span
          class="total"
        >
          R$ 
          0,00
        </span>
      </button>
    </header>
    <div
      class="sc-iqseJM cEOUDX"
    >
      <div
        class="sc-dJjYzT gkmPAG"
      />
    </div>
  </div>
</div>
`;
